src/main/java/org/springframework/integration/dsl/LambdaMessageProcessor.java

```diff
@@ -125,6 +126,9 @@ public Object processMessage(Message<?> message) {
 		try {
 			return this.method.invoke(this.target, args);
 		}
+		catch(InvocationTargetException e) {
+			throw new MessageHandlingException(message, e.getCause());
+		}
```

Comment:
Do you mean in SI or the DSL?

The only usage I see in the DSL is `AmqpPollableMessageChannelSpec.templateChannelTransacted()` which doesn't need unwrapping.